packages/storage-events/src/todos-log.js

```diff
@@ -0,0 +1,120 @@
+module.exports = [
```

Comment:
filename could be a bit more descriptive. maybe `duplicating-todos-log` or something like that?

Reply:
I've changed it to `duplicated-todos-through-move.js` and moved it into `__mocks__`